sources/embedded/embedded_test.go

```diff
@@ -38,7 +38,7 @@ func TestGoEmbed(t *testing.T) {
 	}
 
 	s := Resource(assetNames, func(name string) ([]byte, error) {
-		return assets.ReadFile(filepath.Join("testfiles", name))
+		return assets.ReadFile(strings.Join([]string{"testfiles", name}, "/"))
```

Comment:
This looks really weird, sepcially for a test case, I would be more straight forward. Something like:
```suggestion
		return assets.ReadFile("testfiles/" + name)
```

Reply:
Fair enough 👍